### ISSUE TITLE
Enable Frameless / Hide Title Bar

### ISSUE THREAD

**Issue by rt3norio** [IsmaelMartinez/teams-for-linux#185]
**Is your feature request related to a problem? Please describe.**
I would like to be able to remove the window title bar, so i can have bigger 'work area' when on fullscreen, because my notebook resolution is not great.

**Describe the solution you'd like**
I would like to have a setting or checkbox that on the next app start would remove the title bar.

**Describe alternatives you've considered**
I could press F11 to achieve the same funcionality

**Additional context**
I don't think there is any more information that i can give

Thanks in advance

**Comment by IsmaelMartinez** [comment]
Might be worth trying with Frameless window https://electronjs.org/docs/api/frameless-window#frameless-window

Adding frame: false to this line should do:
https://github.com/IsmaelMartinez/teams-for-linux/blob/develop/app/index.js#L138

To make it configurable, follow the config.webDebug approach from line: https://github.com/IsmaelMartinez/teams-for-linux/blob/develop/app/index.js#L76

That is pulled from the config folder https://github.com/IsmaelMartinez/teams-for-linux/tree/develop/app/config

**Comment by rt3norio** [comment]
I'm gonna try it on this weekend.
With your guidelines, probably will be easy.
Thanks!

**Comment by IsmaelMartinez** [comment]
Let me know if you need more info/help. Might be easier to chat in gitter channel.

**Comment by IsmaelMartinez** [comment]
Updated the links after the refactoring.

**Comment by IsmaelMartinez** [comment]
hey @rt3norio , let me know if you need any extra help. You can always add a PR with where you are and I can guide you a bit more. Thanks!

**Comment by IsmaelMartinez** [comment]
When down the route of the menu option. Electron has a few menu options that can be added and the toggle. 

From here: https://electronjs.org/docs/api/menu-item?q=togglefullscreen#menuitemmenu

I will close this as this fulfils your alternative solution. It should be in snap edge 0.6.4 (building as I write)

**Comment by IsmaelMartinez** [comment]
just pre-released 0.6.6 so people that don't use snap can test it also. 
https://github.com/IsmaelMartinez/teams-for-linux/releases/tag/v0.6.6

**Comment by reformat0r** [comment]
This is still broken for me.
Changing this line
https://github.com/IsmaelMartinez/teams-for-linux/blob/c4cfa3e82c9cd4790bf6cb00cc352bba6c1bdbe2/app/mainAppWindow/browserWindowManager.js#L56
to
```js
frame: false
```
does enable the feature.
The problem seems to be type coercion:
```js
typeof this.config.frame ===  "string"
// and
Boolean("false") === true
```
